### DISCUSSION TITLE
Welcome to chongluadao-extension Discussions!

### DISCUSSION THREAD

**Discussion by 7onez** [7zones/chongluadao-extension#34] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 7onez** [comment]
Hi yes, you can, please share with me your CV/resume and tell me a
little bit about yourself.

Thank you so much.
Best regards,

Hieu Ngo (Hieupc) - Cybersecurity researcher at Viet Nam's NCSC, APWG,
VirusTotal, URLScan and Scamadviser

**Reply by nghiahung-dev** [reply]
Hi Hieu, thank you for your answer.
This is my CV,  [Nguyễn Nghĩa Hùng_CV.pdf](https://github.com/7zones/chongluadao-extension/files/9837706/25_Nguy.n.Nghia.Hung_CV.pdf)

I am trying to understand this project for my graduation thesis in mid-November. I am a beginner dev so I have many problems but I try to improve this project. If I can improve this project, I will create PRs for features. 🙏

**Comment by 7onez** [comment]
Dear Hung,

It's good to hear from you. So please improve any features of the extension
if possible. We really appreciate it.

Hope all is well,

Thank you

Hieu Ngo
